src/Compute/v2/Models/Server.php

```diff
@@ -106,6 +107,10 @@ class Server extends OperatorResource implements
      */
     public function create(array $userOptions): Creatable
     {
+        if (!isset($userOptions['imageId']) && !isset($userOptions['blockDeviceMapping']['uuid'])) {
+            throw new \RuntimeException('Boot-from-volume UUID or image UUID is required');
```

Comment:
also can we change the error to `imageId or blockDeviceMapping.uuid must be set`